ow/launch/common.launch

```diff
@@ -88,7 +91,8 @@
   <node if="$(arg rqt_gui)" pkg="rqt_gui" type="rqt_gui" name="rqt_gui"
     launch-prefix="bash -c 'sleep 5; $0 $@' "
     args="&#45;&#45;perspective-file $(find ow)/config/default.perspective">
-    <remap from="/StereoCamera/left/image_trigger" to="/_original/StereoCamera/left/image_trigger"/> 
+    <!-- <remap from="/ant_pan_position_controller/command" to="/_original/ant_pan_position_controller/command"/>
+    <remap from="/ant_tilt_position_controller/command" to="/_original/ant_tilt_position_controller/command"/>  -->
```

Comment:
These should be completely removed as we are no longer using remapping to perform the remapping
```suggestion
```